src/tests/verify_resources.py

```diff
@@ -436,6 +439,26 @@ def test_14_res_contacts_match(rgs, rgfns, contacts):
 
     return errors
 
+def test_15_FACILITY_SITE_check():
+    # verify the required FACILITY.yaml and SITE.yaml files
+    errors = 0
+
+    groups = glob.glob(_topdir + '/topology/*')
+    for group in groups:
+        for root, dirs, files in os.walk(group):
+            if root == group:
+                if 'FACILITY.yaml' not in files:
```

Comment:
I'm not wild about using `os.walk` for this and then testing membership.

Can we just to a `for facdir in glob.glob("*/")` and test `os.path.exists(facdir + "FACILITY.yaml")`?

And likewise `for sitedir in glob.glob("*/*/")` and test `os.path.exists(sitedir + "SITE.yaml")` ?

Reply:
Thanks, I will make the modification.